aspnet/fundamentals/localization.rst

```diff
@@ -203,7 +203,16 @@ If you only pass in one of the two (``culture`` or ``ui-culture``), the query st
 CookieRequestCultureProvider
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Production apps will often provide a mechanism to set the culture with the ``ASPNET_CULTURE`` cookie. The sample app provides code to set the cookie for French and English cultures.
+Production apps will often provide a mechanism to set the culture with the ASP.NET Core culture cookie. Use the :dn:method:`~Microsoft.AspNetCore.Localization.CookieRequestCultureProvider.MakeCookieValue`  method of the :dn:cls:`~Microsoft.AspNetCore.Localization.CookieRequestCultureProvider` to create a cookie. 
```

Comment:
Is this `:dn:` stuff the normal format? It doesn't show up at least anywhere else in this file is why I ask.

Also, just a nitpick, but I would normally phrase this as "Use the `blah` method to create a cookie.". The namespace it's in is already available from that link right?